h2o-genmodel/src/main/java/hex/genmodel/RowConverterFactory.java

```diff
@@ -0,0 +1,15 @@
+package hex.genmodel;
+
+import hex.genmodel.easy.EasyPredictModelWrapper;
+import hex.genmodel.easy.RowToRawDataConverter;
+
+public interface RowConverterFactory {
+
+    /**
+     * Prefixed by _ as it is an internal API (skipped for validation by Mojoland).
+     * @return A new instance of {@link RowToRawDataConverter}
+     */
+    RowToRawDataConverter _makeRowConverter(CategoricalEncoding categoricalEncoding,
```

Comment:
as MojoModel is the only class implementing this interface, it seems unnecessary and adds an ugly _public method to our API, why do we need this ifc?

Reply:
it's because it's called by `GenModel`

Reply:
Btw this is public but this is not API.